apps/web/src/components/AccountSelectorModal/AccountInfoModal.tsx

```diff
@@ -0,0 +1,70 @@
+import {
+  Box,
+  Button,
+  Heading,
+  ModalBody,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  Text,
+  VStack,
+} from "@chakra-ui/react";
+import { useDynamicDisclosureContext } from "@umami/components";
+import { type Account } from "@umami/core";
+import { type MouseEvent } from "react";
+import { QRCode } from "react-qrcode-logo";
+
+import { ModalCloseButton } from "../ModalCloseButton";
+
+type AccountInfoModalProps = {
+  account: Account;
+};
+
+export const AccountInfoModal = ({ account }: AccountInfoModalProps) => {
+  const { goBack } = useDynamicDisclosureContext();
+
+  const handleCopyAddress = async (event: MouseEvent<HTMLButtonElement>) => {
+    await navigator.clipboard.writeText(account.address.pkh);
+    goBack();
```

Comment:
Should the modal get closed on copying the address? 
On desktop in similar cases we show toast and keep the modal open